Readthrough (#40)
SpecialeBajs/speciale-rapport: article/Experiment/dataset.tex

        Amazon-Book             & 52,643 & 91,599  & 1.74              & 2,984,108    & 99.9381\% \\ \hline
        Yelp2020                & 24,384 & 20,091  & 0.824             & 594,196      & 99.8787\% \\ \hline
        Amazon-Cell-Sport       & 4,998  & 36,719  & 7.347             & 103,000      & 99.9438\% \\ \hline
        Amazon-Cell-Electronic  & 3,325  & 57,925  & 17.42             & 172,611      & 99.9103\% \\ \hline
        Amazon-Cloth-Electronic & 15,761 & 105,174 & 6.673             & 363,474      & 99,9780\% \\ \hline
        Amazon-Cloth-Sport      & 9,928  & 73,613  & 7.414             & 200,297      & 99,9726\% \\ \hline
    \end{tabular}
    \caption{Comparisons on the datasets}
    \label{tab:dataset-comparison}
\end{table*}

\subsubsection{Interactions in datasets}
As we expect there is a relation between the number of connections for each node and the performance of a convolution layer, we analyzed the different datasets used for our test.
On \autoref{tab:dataset-item-and-user-splits} and \autoref{tab:node-degrees-cell-sport-electronic} the amount of users and items that have a certain amount of interactions can be seen.

> The users in \textit{Amazon-Book} has an average connection of 45.22 connections and items have an average of 25.99, which makes it the dataset with the largest average degree of nodes compared to the other datasets.

Reviewer: ```suggestion
Users in \textit{Amazon-Book} have an average ndoe degree of 45.22 and items have an average node degree of 25.99 connections, which makes it the dataset with the highest average node degree compared to the other datasets.
```

> For Yelp2020 the average degree of nodes are 19.04 for users and 23.31 for items.

Reviewer: ```suggestion
For Yelp2020 the average node degree is 19.04 for users and 23.31 for items.
```

> The other four amazon datasets have in common that the average degree of item nodes is small.

Reviewer: ```suggestion
The other four amazon datasets have in common that the average node degree for items is small.
```

> User nodes in \textit{Amazon-Cell-Sport} have an average degree of 17.07 and items have a degree of 2.58.

Reviewer: ```suggestion
Users in \textit{Amazon-Cell-Sport} have an average node degree of 17.07 and items have an average node degree of 2.58.
```

> \textit{Amazon-Cell-Electronic} has an average degree of 42.22 for user nodes and 2.73 for item nodes.

Reviewer: ```suggestion
\textit{Amazon-Cell-Electronic} has an average node degree of 42.22 for users and an average node degree 2.73 for items.
```

> \textit{Amazon-Cloth-Electronic} has an average degree of 19.04 for users and 3.12 for items.

Reviewer: ```suggestion
\textit{Amazon-Cloth-Electronic} has an average node degree of 19.04 for users and an avevrage node degree of 3.12 for items.
```

> \textit{Amazon-Cloth-Sport} has an average degree of 16.72 for users and 2.49.

Reviewer: ```suggestion
\textit{Amazon-Cloth-Sport} has an average node degree of 16.72 for users and an average node degree of 2.49 for items.
```

        300+    & 7                             & 0.028 \%                               & 61                               & 0.30 \%                    & 1                          & 0.02 \%                    & 0     & 0  \%        & 500   & 0.94 \%  & 271   & 0.29 \%  \\ \hline
        Avg con & \multicolumn{2}{l|}{19.04}    & \multicolumn{2}{l|}{23.31}             & \multicolumn{2}{l|}{17.07}       & \multicolumn{2}{l|}{2.58}  & \multicolumn{2}{l|}{45.22} & \multicolumn{2}{l|}{25.99}                                                              \\ \hline
    \end{tabular}

>     \caption{Amount of nodes within a certain node degree for Amazon-Cell-Sport, Amazon-Book and Yelp2020. The Avg connection shows how many connections each user or item have in average}

Reviewer: ```suggestion
    \caption{Amount of nodes within a certain node degree for Amazon-Cell-Sport, Amazon-Book and Yelp2020. The Avg connection shows how many connections each user or item have on average}
```
